CriticalMass/RequestManager.swift

```diff
@@ -128,3 +132,18 @@ public class RequestManager {
         }
     }
 }
+
```

Comment:
The extensions also needs to be marked as @available(macOS 10.12, *) for spm compatibility

Reply:
fixed